facia/conf/routes

```diff
@@ -6,8 +6,8 @@
 # For dev machines
 GET        /assets/*path                                                            dev.DevAssetsController.at(path)
 
-GET        /_healthcheck                                                            conf.HealthCheck.healthcheck()
-GET        /_fronts_cdn_healthcheck                                                 conf.HealthCheck.cdnHealthcheck()
+GET        /_healthcheck                                                            conf.HealthCheckController.healthCheck()
+GET        /_fronts_cdn_healthcheck                                                 conf.HealthCheckController.healthCheck()
```

Comment:
Since we are caching result, we can use the same endpoint for normal and cdn healthcheck.
Eventually we can remove `/_fronts_cdn_healthcheck ` all together